lib/src/execute.rs

```diff
@@ -276,11 +277,15 @@ impl ExecuteCtx {
             .expect("`memory` is exported")
             .size(&store);
 
+        let distant = Instant::now().duration_since(now);
```

Comment:
...and similarly, a slight name tweak

```suggestion
        let request_duration = Instant::now().duration_since(start_timestamp);
```